pkg/hyper/helper.go

```diff
@@ -187,13 +187,18 @@ func buildLabelsWithAnnotations(labels, annotations map[string]string) map[strin
 		return labels
 	}
 
+	newLables := make(map[string]string)
+	if labels != nil {
+		newLables = labels
```

Comment:
nit: it's better to exchange the position between `newLables = labels` and `make(map[string]string)`